draft-ietf-sidrops-aspa-notation-02.md

```diff
@@ -118,21 +131,34 @@ This field represents a Provider AS as defined in section 3.3 of
 
 ## asn
 
-This field can optionally be prepended with the string "AS" followed by
-a decimal value of a 32 bit Autonomous System Number using the asplain
-presentation as specified in [@!RFC5396]. Decimal values MUST be used,
-and values MUST be part of the range 0-4294967295.
+This field consists of the string "AS" followed by a decimal value of a 32-bit
+Autonomous System Number using the asplain presentation as specified in
+[@!RFC5396]. Decimal values MUST represent a 32 bit value, and therefore MUST
+be part of the range 0-4294967295.
 
 # Example Notations
 
+Some example notations are listed below. The last example is not advised for
+readability but is technically allowed by this specification.
+
 ~~~
 AS65000 => AS65001
-AS65000 => AS65002
-AS65000 => AS65001, AS65002, AS65003
-
-65000 => 65001
+65000 => AS65001
 65000 => AS65002
-65000 => 65001, 65002, 65003
+AS65000 => AS65001, AS65002,AS65003
+
+AS65000 => [ AS65001, AS65002, AS65003 ]
+
+AS65000 => [
+    AS65001,
+    AS65002,
+    AS65003
+]
+
+AS65000 => [AS65001,
+                     65002
```

Comment:
No longer allowed?

Reply:
Indeed - this should be AS65002

But, I will merge this first and then make a new branch for -03 and fix it there.